wisembly/amqp-bundle/1.3/Makefile

```diff
@@ -0,0 +1,16 @@
+CONSOLE=bin/console
+
+ifndef GATE
+gate:
+	$(error GATE not set)
+endif
+
+gate:
+
+consume: gate $(CONSOLE)
+	$(CONSOLE) wisembly:amqp:consume $(GATE) $(OPTIONS)
+
+publish: gate $(CONSOLE)
+	$(CONSOLE) wisembly:amqp:publish $(GATE) "$(MESSAGE)"
```

Comment:
well, this is also why we don't want to proxy all commands through the Makefile: we loose the possibility to use options to refine the command (and so people still need to know the actual command to use it with options or to see the help of the command, which relies on passing the `-h` option).
We provide Makefile commands for tasks which involve multiple Symfony commands, or additional command not being part of the Symfony console